web/modules/custom/do_base/tests/src/Traits/ReflectionTrait.php

```
@@ -0,0 +1,85 @@
+   * Get protected value from the object.
+   *
+   * @param object $object
+   *   Object to set the value on.
+   * @param string $property
+   *   Property name to get the value. Property should exists in the object.
+   *
+   * @return mixed
+   *   Protected property value.
+   */
+  protected static function getProtectedValue($object, $property): mixed {
+    $class = new \ReflectionClass($object::class);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '79', column '18'). (undefined)

(MissingImport)